my_class.rb

```diff
@@ -0,0 +1,12 @@
+require 'MyEnumerable'
+
+class MyList
+  include MyEnumerable
+  def initialize(*list)
```

Comment:
- [ ] `#initialize` method should accept multiple arguments. Thanks to that, calling the constructor with a list of numbers as in the test code is possible `MyList.new(1, 2, 3, 4)`
```ruby
    def initialize(*args)
        @list = []
        args.each { |arg| @list << arg }
    end
```

Reply:
> * [ ]  `#initialize` method should accept multiple arguments. Thanks to that, calling the constructor with a list of numbers as in the test code is possible `MyList.new(1, 2, 3, 4)`
> 
> ```ruby
>     def initialize(*args)
>         @list = []
>         args.each { |arg| @list << arg }
>     end
> ```

### That is exactly what my code is doing:

<img width="259" alt="Screenshot 2023-03-10 at 9 24 08 PM" src="https://user-images.githubusercontent.com/110017001/224369380-4f291064-cf9d-4a09-b168-85fb5f9028b5.png">

### All the test cases are working:

<img width="450" alt="Screenshot 2023-03-10 at 9 31 14 PM" src="https://user-images.githubusercontent.com/110017001/224370836-58b108fb-9fe7-4b8a-853c-66c45cf7a912.png">